core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.component.spec.ts

```diff
@@ -481,7 +481,7 @@ describe('Skills List Component', () => {
     ).toHaveBeenCalledWith(true);
     expect(alertsService.addSuccessMessage).toHaveBeenCalledWith(
       'The skill has been assigned to the topic.',
-      1000
+      5000
```

Comment:
Is this just for debugging or 5000ms is needed for this function? Just to make sure code that is for debugging is included when merging.

Reply:
No, actually, the a toast/tooltip message appears on assigning or unassigning a topic.. However, 1 second is extremely low to even see the complete message.. so updating it to 5 second. (Also, useful for fetching the element.)

Reply:
Can we avoid adding a custom timeout in the first place and stick to the default one?

Reply:
@Vir-8 
Default 1.5s, fine to keep it this much? Maybe we should keep it at least 3 secs?

Reply:
@Akhilesh-max Okay, let's make it 3 seconds

Reply:
@Vir-8 Done.